genai-perf/genai_perf/parser.py

```diff
@@ -135,6 +135,9 @@ def _check_conditional_args(
     Check for conditional args and raise an error if they are not set.
     """
 
+    if args.num_prompts <= 0:
```

Comment:
Thanks for adding this check. Can you use ArgParse instead to make sure this arg is positive?

Reply:
modified to use argparse for checking input values